utils/centos7-ci.sh

```diff
@@ -27,7 +27,7 @@ install_dependencies() {
 
     # install openresty
     yum install -y yum-utils && yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
-    yum install -y openresty-debug openresty-openssl111-debug-devel
+    yum install -y openresty openresty-debug openresty-openssl111-debug-devel
```

Comment:
Why install `openresty`?

Reply:
Rpm package requires [-d 'openresty >= 1.17.8.2'](https://github.com/api7/apisix-build-tools/blob/master/Makefile#L45). It seems only works for `openresty` but not `openresty-debug`

Reply:
Better to add a comment for it.

Reply:
fixed. PTAL